### PR TITLE
[openjpeg] Update to 2.5.0. JB#58622

### PR DESCRIPTION
**Pull request by mlehtima**

### CODE REVIEW

File: rpm/openjpeg.spec [hunk 1]
```diff
@@ -68,7 +68,7 @@ popd
 
 %files devel
 %defattr(-,root,root,-)
-%{_includedir}/openjpeg-2.4/
```
Comment (Thaodan) [inline rpm/openjpeg.spec:71]: Can you replace that be defining a macro and then using it:
```spec
%global version_major_minor %{lua: a=rpm.expand('%{version}')\
                                   print(string.match(a, "%d%.%d"))}
```
Doing so removes the need to replace anything besides the version when here's a new version.

Comment (mlehtima) [reply rpm/openjpeg.spec:71]: Probably even easier would be to use wildcard in the file section

File: rpm/openjpeg.spec [hunk 2]
```diff
@@ -68,7 +68,7 @@ popd
...
+%{_includedir}/openjpeg-*/
 %{_libdir}/pkgconfig
 %{_libdir}/libopenjp2.so
```
Comment (Thaodan) [inline rpm/openjpeg.spec:73]: Wildcard here too?

**Review by Thaodan — APPROVED** [review]
LGTM

File: rpm/openjpeg.spec (hunk 2 above)
Comment (mlehtima) [reply rpm/openjpeg.spec:73]: That 2 in jp2 is not related to version but instead refers to JPEG 2000.

Comment (Thaodan) [reply rpm/openjpeg.spec:73]: ok

**Review by mkosola — APPROVED** [review]

**Pull request merged**